backend/src/main/java/com/greenfoxacademy/backend/services/UserServiceImpl.java

```diff
@@ -0,0 +1,16 @@
+package com.greenfoxacademy.backend.services;
+
+import com.greenfoxacademy.backend.models.User;
+import com.greenfoxacademy.backend.repositories.UserRepository;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserServiceImpl implements UserService {
+    UserRepository userRepository;
```

Comment:
Shouldn't this be private and maybe final too?
When is is going to be created, should't we go with dependency inhjection within constructors?